app/src/lang/index.ts

```diff
@@ -0,0 +1,115 @@
+import {defineMessages} from "react-intl.macro";
+
+const responses = defineMessages({
+    recipeDeletedSuccess: {
+        id: 'recipe.deleted.success',
```

Comment:
For ids we use uuids. I think this is that the ids have to be globally unique across the whole front end app.

```bash
uuid() {
    local uuid=$(uuidgen | awk '{print tolower($0)}' | tr -d '\n')
    echo $uuid | pbcopy
    echo $uuid
}
```
I added this to my zshrc so I could just call uuid and have one copied to my clipboard as I was using it so much.